Project List/scripts.js

```diff
@@ -0,0 +1,15 @@
+const hide_buttons = document.getElementsByClassName("btn");
```

Comment:
the name of variable should be in camelCase. `hide_button => hideButton`

Reply:
Solved in the latest merge